src/osd/OSD.cc

```diff
@@ -9030,7 +9030,8 @@ void OSD::process_peering_events(
 {
   bool need_up_thru = false;
   epoch_t same_interval_since = 0;
-  OSDMapRef curmap = service.get_osdmap();
+  assert(!pgs.empty());
```

Comment:
and also put more details into the commit message: `clean up` is too vague a word, IMHO.